.gitignore

```diff
@@ -21,18 +22,4 @@ distribution/openhabhome/userdata/mapdb/storage.mapdb
 distribution/openhabhome/userdata/mapdb/storage.mapdb.p
 distribution/openhabhome/userdata/mapdb/storage.mapdb.t
 features/*/src/main/history
-addons/voice/org.openhab.voice.marytts/lib/emotionml-checker-java-1.0-SNAPSHOT.jar
-addons/voice/org.openhab.voice.marytts/lib/freetts-1.0.jar
-addons/voice/org.openhab.voice.marytts/lib/freetts-de-1.0.jar
-addons/voice/org.openhab.voice.marytts/lib/freetts-en_us-1.0.jar
-addons/voice/org.openhab.voice.marytts/lib/jtok-core-1.9.1.jar
-addons/voice/org.openhab.voice.marytts/lib/marytts-lang-de-5.1-SNAPSHOT.jar
-addons/voice/org.openhab.voice.marytts/lib/marytts-lang-en-5.1-SNAPSHOT.jar
-addons/voice/org.openhab.voice.marytts/lib/mwdumper-1.16.jar
-addons/voice/org.openhab.voice.marytts/lib/opennlp-maxent-3.0.1-incubating.jar
-addons/voice/org.openhab.voice.marytts/lib/opennlp-tools-1.5.1-incubating.jar
-addons/voice/org.openhab.voice.marytts/lib/sgt-3.0.jar
-addons/voice/org.openhab.voice.marytts/lib/voice-bits1-hsmm-5.0-SNAPSHOT.jar
-addons/voice/org.openhab.voice.marytts/lib/voice-bits3-hsmm-5.0-SNAPSHOT.jar
-addons/voice/org.openhab.voice.marytts/lib/voice-cmu-slt-hsmm-5.0.jar
```

Comment:
Why is this removed? The binding is still there and the JARs are still download during the build.

Reply:
I though marytts is already ported, because of that line in the same file: `bundles/io/org.openhab.io.multimedia.tts.marytts/lib/` but I know realize that the "io/" part renders this line ineffective anyway